tools/manage-osqueryd.ps1

```diff
@@ -6,7 +6,7 @@
 #  of patent rights can be found in the PATENTS file in the same directory.
 
 param(
-  [string] $args = "",
+  [string] $startup_args = "",
```

Comment:
Would you mind, making this `camlCase` instead of `snake_case` just for consistency? <3